tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -353,6 +353,17 @@ def testee(a: cases.IFloatField) -> gtx.Field[[IDim], bool]:
     )
 
 
+@pytest.mark.parametrize("inp", [0.0, 2.0])
+def test_astype_bool_scalar(cartesian_case, inp):  # noqa: F811 # fixtures
+    @gtx.field_operator
+    def testee(inp: float) -> gtx.Field[[IDim], bool]:
+        return broadcast(astype(inp, bool), (IDim,))
+
+    out = cases.allocate(cartesian_case, testee, cases.RETURN)()
+
+    cases.verify(cartesian_case, testee, inp, out=out, ref=bool(inp))
+
+
 def test_astype_float(cartesian_case):  # noqa: F811 # fixtures
```

Comment:
can we parametrize this test and `test_astype_bool_field` above?

Reply:
As discussed for simplicity this is not done.